File: src/libs/ThirdParty.VBSuppliers/Properties/ExpediaRapid/SerializableClasses/Rate.vb
Imports Newtonsoft.Json
Namespace ExpediaRapid.SerializableClasses
    Public Class Rate


        <JsonProperty("type")>
        Public Property RateType As String

        <JsonProperty("value")>
        Public Property Amount As Decimal

        <JsonProperty("currency")>
        Public Property CurrencyCode As String


    End Class
End Namespace

File: src/libs/iVectorOne.Suppliers/ExpediaRapid/SerializableClasses/Rate.cs
namespace iVectorOne.Suppliers.ExpediaRapid.SerializableClasses
{
    using Newtonsoft.Json;

    public class Rate
    {


        [JsonProperty("type")]
        public string RateType { get; set; }

        [JsonProperty("value")]
        public decimal Amount { get; set; }

        [JsonProperty("currency")]
        public string CurrencyCode { get; set; }


    }
}